clevertap-core/src/main/java/com/clevertap/android/sdk/DeviceInfo.java

```diff
@@ -377,6 +308,33 @@ private double toTwoPlaces(double n) {
         }
     }
 
+    /**
+     * Placeholder for data
+     */
+    private static class WindowSize {
+        public final int localDpi;
+        public final double doubleWidth;
+        public final double doubleHeight;
+
+        public WindowSize(int localDpi, double doubleWidth, double doubleHeight) {
```

Comment:
Can this be private?

Reply:
this is a constructor and hence public to be used within same class.

Reply:
Access will also be possible if private since it is an inner class?